FAQ.md

```diff
@@ -1,14 +1,70 @@
 # Frequently Asked Questions
 
-1. [How do I fix invalid entitlements?  ](#how-do-i-fix-invalid-entitlements)
+1. [How do I declare a new Haskell dependency?](#how-do-i-declare-a-new-haskell-dependency)
+1. [How do I add or override Haskell dependencies in the package set?](#how-do-i-add-or-override-haskell-dependencies-in-the-package-set)
+1. [How do I extend my Obelisk application with more local packages?](#how-do-i-extend-my-obelisk-application-with-more-local-packages)
+1. [How do I use `ob run` over HTTPS?](#how-do-i-use-ob-run-over-https)
+1. [How do I fix invalid entitlements?](#how-do-i-fix-invalid-entitlements)
 1. [`ob thunk update` or `ob deploy update` fails](#ob-thunk-update-or-ob-deploy-update-fails)
 1. [How do I fix `Ambiguous module name` errors?](#how-do-i-fix-ambiguous-module-name-errors)
-1. [What does `ob run` actually do?](#what-does-ob-run-actually-do)
+
+### How do I declare a new Haskell dependency?
+
+Every component of your Obelisk application is a standard [cabal](https://www.haskell.org/cabal/) package. That means declaring new Haskell dependencies simply involves updating the `build-depends` field in the appropriate cabal file (`backend.cabal` for example). By default, Obelisk will use its curated package set to choose which version of each package you get. It's possible that the package you need is not already in the curated set or the curated version isn't the one you want. See [How do I add or override Haskell dependencies in the package set?](#how-do-i-add-or-override-haskell-dependencies-in-the-package-set) for a solution to this.
+
+### How do I add or override Haskell dependencies in the package set?
+
+To add a Haskell package that doesn't exist in Obelisk's package set or to override the version of a package, use the `overrides` attribute in your project's `default.nix`. For example, to use a specific version of the `aeson` package fetched from GitHub and a specific version of the `waargonaut` package fetched from Hackage, your `default.nix` will look like:
```

Comment:
```suggestion
To add an Haskell package that doesn't exist in Obelisk's package set or to override the version of a package, use the `overrides` attribute in your project's `default.nix`. For example, to use a specific version of the `aeson` package fetched from GitHub and a specific version of the `waargonaut` package fetched from Hackage, your `default.nix` will look like:
```

Reply:
This is locale-dependent. It's more natural in American English to use "a Haskell".